docs/HTTPClient.md

```diff
@@ -0,0 +1,71 @@
+# Call the other services on the Platform project
+You can call any service or any endpoints defined on the Platform project, obtaining and using a proxy object. 
+
+For example, if you need to connect to a CRUD, you have to use a Proxy towards the crud-service.
+
+You can get a proxy calling these methods booth on `Request`(the first argument of handler) and `Service` (the Fastify instance):
+
+* `getServiceProxy(options)` - returns a proxy  passing through the [Microservice Gateway](https://docs.mia-platform.eu/runtime_suite/microservice-gateway/).  
+    * `options` - is an object with the following optional fields:
+        * `port` - an integer that identifies the port of the service to be queried.
+        * `protocol` - a string that identifies the protocol to use (only `http` and `https` are supported, the default value is `http`).
+        * `headers` - an object that represents the set of headers to forward to the service.
+        * `prefix` - a string representing the prefix of the service call path.
+* `getDirectServiceProxy(serviceName, options)` - returns a direct proxy to the service.
+    *  `serviceName` - The name of the service to call. You can't specify the port here, you have to do in `options.port`.
+     * `options` - The same options described above.
+
+Potentially, the `getDirectServiceProxy` method allows you to also query services outside the Platform. In this case, however, it is necessary to bear in mind that the platform headers will be automatically forwarded.
+
+Both proxies, by default, forward the four Mia headers to the service called. In addition, other headers of the original request can also be forwarded to the named service. To do this it is necessary to define an additional environment variable, `ADDITIONAL_HEADERS_TO_PROXY`, whose value must be a string containing the keys of the headers to be forwarded separated by a comma.
+
+Both proxies expose the methods to perform a specific HTTP request to service.
+
+ * `get(path, querystring, options)`
+ * `post(path, body, querystring, options)`
+ * `put(path, body, querystring, options)`
+ * `patch(path, body, querystring, options)`
+ * `delete(path, body, querystring, options)`
+
+The params to be passed to these functions are:
+
+ * `path` -  a string that identifies the route to which you want to send the request.
+ * `body` - optional, the body of the request which can be:
+    * a JSON object
+    * a [Buffer](https://nodejs.org/api/buffer.html#)
+    * one [Stream](https://nodejs.org/api/stream.html)
+ * `querystring` - optional, an object that represents the querystring.
```

Comment:
```suggestion
 * `querystring` - optional, an object that represents the querystring;
```

Reply:
I don't understand why using ";" in this case. Can you explain to me?